index.html

```diff
@@ -1,39 +1,204 @@
+          I’m a software developer! I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="social-media-button">
+          <div class="social-media">
+            <p>LET’S CONNECT</p>
+            <ul class="social-icon pl-1">
+              <li class="social-item"><a href="https://twitter.com/HenaryKhuraijam"><img src="images/twitter.png"
+                    alt="twitter"></a></li>
+              <li class="social-item"><a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/"><img
+                    src="images/linkedin.png" alt="linkedin"></a></li>
+              <li class="social-item"><a href="https://medium.com/"><img src="images/medium.png" alt="medium"></a></li>
+              <li class="social-item"><a href="https://github.com/HenaryKhuraijam"><img src="images/github.png"
+                    alt="github"></a></li>
+              <li class="social-item"><a href="https://angel.co/"><img src="images/angellist.png" alt="angellist"></a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">Get my resume</button>
+        </div>
+      </div>
+      <ul class="projects">
+        <li class="flex skills-dropdown">
+          <div class="flex">
+            <div>Languages</div><img class="icons" src="icons/drop.png" alt="menu-icn">
+          </div>
+          <ul class="language-ul">
+            <li id="lang-item1"><img src="images/js.png" alt="twitter link">JavaSript</li>
+            <li id="lang-item2"><img src="images/html.png" alt="linkedin link">HTML</li>
+            <li id="lang-item3"><img src="images/css.png" alt="medium link">CSS</li>
+          </ul>
+        </li>
+        <li class="flex skills-dropdown" id="frameworks">
+          <div>Frameworks</div><img class="icons" src="icons/open.png" alt="menu-icn">
+        </li>
+        <li class="flex skills-dropdown" id="skills">
+          <div>Skills</div><img class="icons" src="icons/open.png" alt="menu-icn">
+        </li>
+      </ul>
+    </section>
   </main>
-  <footer></footer>
+  <footer class="footer" id="contacts">
+    <div class="content-placeholder" id="contact-placeholder">
+      <h2 class="head-1" id="contact-me">Contact Me</h2>
+      <p id="contact-msg">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>    
+      <form class="form flex" action="https://formspree.io/f/mqkoalvp"
+      method="POST"
+      enctype="multipart/form-data">
+        <input class="form-input input-line" type="text" placeholder="Enter your name" name="name" maxlength="30" required/>
+        <input class="form-input input-line" type="email" id="email" name="email" placeholder="yourid@gmail.com" required/>
```

Comment:
Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
    font-size: 17px;
    line-height: 20px;
```